tests/client/feedback/test_dataset.py

```diff
@@ -554,7 +584,7 @@ async def test_update_dataset_records_in_argilla(
                 },
             ]
         )
-    with pytest.warns(UserWarning, match="if you are trying to set `suggestions` directly"):
+    with pytest.raises(TypeError, match='"suggestions" has allow_mutation set to False and cannot be assigned'):
```

Comment:
```suggestion
    with pytest.raises(TypeError, match="\"suggestions\" has allow_mutation set to False and cannot be assigned"):
```

Reply:
Didn't try this, but I assume we should keep the standard for using double-quotes and just scape the nested ones